package.json

```diff
@@ -48,9 +54,7 @@
   "jspm": {
     "name": "single-spa",
     "main": "lib/single-spa.js",
-    "dependencies": {
-      "custom-event": "^1.0.0"
```

Comment:
and custom-event no longer needs to be a jspm dependency. We still need jspm for the tests, though